_includes/layout.11ty.js

```diff
@@ -30,13 +30,19 @@ module.exports = async function(data) {
 		</ul>`;
 	}
 
+	let meta_description = `A read-only indieweb self-hosted archive of${ data.pagination && data.pagination.hrefs && data.pagination.hrefs.length ? ` all ${data.pagination.hrefs.length}` : ""} of ${data.metadata.username}’s tweets.`;
+	if (data.page.fileSlug === "tweet-pages" && data.tweet && data.tweet.full_text) {
+		// note that data.tweet.full_text is already HTML-escaped
+		meta_description = data.tweet.full_text.replace(/\s+/g, " ");
```

Comment:
Q: Does Twitter capture multiple sequential spaces in any meaningful way? If so, do they come across as `&nbsp;` or something else? Just wondering if this might cause issues. Also, some people do use returns to format tweets. Does `full_text` already have those removed?

Reply:
Returns do format tweets, but I'm not at all convinced that everything can read a meta description with carriage returns in it. (Are carriage returns in an HTML attribute even valid?) So I pulled them out so that the meta description is a single line. (Dealing with multiple spaces as a side effect.) Open to suggestions here.

Reply:
I think removing all spaces makes sense for this context.